src/styles/NumericResultsStyles.ts

```diff
@@ -6,15 +6,31 @@ export const NumericResultsUl = styled.ul`
 	display: flex;
 	flex-direction: column;
 	flex-wrap: no-wrap;
-	gap: 3vh;
+	gap: 2vh;
 	justify-content: flex-start;
 	align-items: flex-start;	
 	width: 90%
 	height: 80vh;
+	padding: 1vh 1vw;
 `;
 
-export const HeaderInfo = styled.p`
+export const HeaderInfo1 = styled.p`
 	font-weight: bold;
+	margin: 15px 5px 15px 0px;
+`;
+
+export const CustomParagraph1 = styled.p`
+	font-weight: normal;
+	margin: 15px 15px 15px 0px;
+`;
+export const HeaderInfo2 = styled.p`
+	font-weight: bold;
+	margin: 5px 5px 15px 5px;
+`;
+
+export const CustomParagraph2 = styled.p`
+	font-weight: normal;
+	margin: 5px 15px 15px 0px;
```

Comment:
jak te dwa paragrafy różnią się tylko jedną rzeczą to możesz je zrobic jako jeden z propsami, np. 

`export type CPProps = {
bottomMargin: int;
}
`
 i wtedy robisz 
`export const CustomParagraph = styled.p<CPProps>'
    font-weight: normal;
    margin: 5px 15px 15px ${(props) => props.bottomMargin}px;
';`
a w jsx'ie:
`<CustomParagraph bottomMargin={5}>tekst</ CustomParagraph>`